src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,388 @@ tree_item_index: 1100
+
+The policy attached to this role can be added as shown in this **sample**:
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucket",
+        "s3:GetObject",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": [
+        "arn:aws:s3:::bucket-name",
+        "arn:aws:s3:::bucket-name/\*"
+      ]
+    }
+  ]
+}
+```
+
+The policy for **Upload** must be able to put and get objects from the S3 bucket, **for example**:
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+  {
+    "Sid": "VisualEditor0",
+    "Effect": "Allow",
+    "Action": [
+    "s3:PutObject",
+    "s3:GetObject",
+        "s3:AbortMultipartUpload",
+        "s3:DeleteObject",
+        "s3:ListMultipartUploadParts"
+      ],
+      "Resource": "arn:aws:s3:::bucket-name/*"
+    },
+    {
+      "Sid": "VisualEditor1",
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucketMultipartUploads",
+        "s3:ListBucket",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": "arn:aws:s3:::bucket-name"
+    }
+  ]
+}
+```
+
+#### Nuxeo Configuration
+
+##### Transient Store on AWS
+
+To use S3 direct upload with Nuxeo, you will need to add another policy to a new role **for example**:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListAllMyBuckets"
+            ],
+            "Resource": "arn:aws:s3:::*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetBucketLocation",
+                "s3:AbortMultipartUpload",
+                "s3:ListMultipartUploadParts",
+                "s3:ListBucketMultipartUploads"
+            ],
+            "Resource": "arn:aws:s3:::yourbucketname"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObject",
+                "s3:AbortMultipartUpload",
+                "s3:ListMultipartUploadParts",
+                "s3:ListBucketMultipartUploads"
+            ],
+            "Resource": "arn:aws:s3:::yourbucketname/*"
+        }
+    ]
+}
+```
+
+Make sure that all configuration values are set in the `nuxeo.conf`:
+
+```
+nuxeo.s3storage.transient.bucket=
+nuxeo.aws.accessKeyId=
+nuxeo.aws.secretKey=
+nuxeo.aws.region=
+nuxeo.s3storage.bucket=
+```
+{{#> callout type='note'}}
+
+Since the work in [NXP-29548](https://jira.nuxeo.com/browse/NXP-29548) 'nuxeo.s3storage.useDirectUpload=true' and 'nuxeo.s3storage.transient.roleArn=' (documented as part of the previous version configuration) are no longer necessary **for the Nuxeo Aspera Connector** and can be removed, **if you only use these values for the Nuxeo Aspera Connector, and not for any other reason**.  See upgrade notes.
```

Comment:
Maybe it deserves a link also to the upgrade notes. WDYT?
BTW, please be aware that the ticket **does not state** any upgrade notes (cc @RSalem07)

Reply:
PS: @lisamcintyre , this ticket is an internal one, so it won't be accessible 🤷🏼‍♂️

Reply:
@nmpcunha will work through the links (concerning the upgrade page AND the fact this ticket isn't viewable).  will see what makes the most sense to point to for information.  Thanks for pointing that out 👍